src/bundle/Resources/views/themes/admin/ui/search/list.html.twig

```diff
@@ -1,3 +1,7 @@
+{#
+    This view is deprecated and will be removed in the next major version.
```

Comment:
We should remove it in this merge up or separate PR?

Reply:
> We should remove it in this merge up or separate PR?

Dropped in a separate commit dae1c42 since this PR is going to be merged in a fast-forward way.

@ezsystems/documentation-team: doc needed:
eZ Design admin theme template (view, depending on doc convention) `@ezdesign/ui/search/list.html.twig` has been removed. Use `@ezdesign/ui/search/index.html.twig` instead.